functionary/core/api/v1/views/task.py

```diff
@@ -67,8 +80,22 @@ def get_serializer_class(self):
         },
         parameters=HEADER_PARAMETERS,
     )
-    def create(self, request, *args, **kwargs):
-        request_serializer = self.get_serializer(data=request.data)
+    def create(self, request: Request, *args, **kwargs):
+        if not request.data.get("parameters"):
+            request.data["parameters"] = {}
+        else:
+            request.data["parameters"] = json.loads(request.data["parameters"])
```

Comment:
This will throw an exception when the content type is `application/json` because the data is already a dict.  As mentioned elsewhere, I'm okay with making this endpoint support `multipart/form-data` only, but if we're going to do that we need to update it to actually do that so it will return an appropriate response if someone tries to submit as `application/json`.  Also, the documentation needs to updated to reflect the supported content-type(s).

And with all that said, it seems like you it should be possible to have the serializers handle this for you rather than calling `json.loads` manually.

Reply:
Fixed. The `json.loads` and `json.dumps` now only gets run if `request.FILES` is not `None`